### PR TITLE
Mark test_dns as flaky

### PR DESCRIPTION
**Pull request by Hasan6979**
### Problem
*--describe problem being solved--*

### Solution
*--describe selected solution--*

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] changelog.md is updated

### CODE REVIEW

**Review by gytsto — APPROVED** [review]

File: nat-lab/tests/test_dns.py [hunk 1]
```diff
@@ -28,6 +28,7 @@
         pytest.param(DNS_IPV6_SERVER_ADDRESS, id="IPv6"),
     ],
 )
+@pytest.mark.xfail(reason="Test is flaky - JIRA issue LLT-4242")
```
Comment (stalowyjez) [inline nat-lab/tests/test_dns.py:31]: Maybe I overlooked something, but I see this test failing only once and only for IPv6 setting, so maybe we could mark only IPv6 part, or even wait a bit if this failure reoccurs?

Comment (Hasan6979) [reply nat-lab/tests/test_dns.py:31]: We could see this entire week and then decide ?

**Review by Jauler — APPROVED** [review]
+1.0

**Comment by Jauler** [comment]
Lets close this pull request and cancel the jira task because it seems like this flakyness is not really flakyness-enough :D

**Pull request closed without merging**